clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/BlobTransferUtils.java

```diff
@@ -1,12 +1,17 @@
 package com.linkedin.davinci.blobtransfer;
 
 import io.netty.handler.codec.http.HttpResponse;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
 
 
 public class BlobTransferUtils {
   public static final String BLOB_TRANSFER_STATUS = "X-Blob-Transfer-Status";
   public static final String BLOB_TRANSFER_COMPLETED = "Completed";
   public static final String BLOB_TRANSFER_TYPE = "X-Blob-Transfer-Type";
+  public static final long BLOB_TRANSFER_DISABLED_OFFSET_LAG_THRESHOLD = 1000L;
```

Comment:
1000 is too small as we usually can do a few k records ingestion within a minute for single topic-partition; Or at least let's make it configurable as mentioned below.

Reply:
Changed the default lag to 100000L and make this value configurable.